libs/react-components/community/src/components/form/file-upload/file-upload.tsx

```diff
@@ -70,20 +70,33 @@ export interface FileUploadProps extends FormLabelProps {
    */
   disabled?: boolean;
   /**
-   * Size limit in bytes
+   * Size limit in megabytes
    */
-  sizeLimit?: number;
+  maxSize?: number;
   /**
    * Callback to be used when files are rejected due to size
    */
   onInvalidSize?: (files: File[]) => void;
 }
 
+const getDefaultHelpers = (
+  { accept, maxSize }: Partial<FileUploadProps>,
+  getLabel: ILabelContext['getLabel']
+): FormHelperProps | undefined => {
+  const text = [
+    accept && `${getLabel('file-upload.accept')} ${accept.replace(',', ', ')}`,
+    maxSize && `${getLabel('file-upload.max-size')} ${maxSize}MB`,
+  ]
+    .filter(Boolean)
+    .join(', ');
```

Comment:
Because both translation start with capital letter, we should separate them with dot not comma.

Reply:
Changed separator.